documentation/user_doc.md

```diff
@@ -0,0 +1,55 @@
+# Nexus Replication Plugin
+
+[https://github.com/griddynamics/nexus-replication-plugin](https://github.com/griddynamics/nexus-replication-plugin)
+
+This is Nexus plugin that provides replication logic when new (Maven) artifacts are stored in a hosted repository. 
+Basically it POST an HTTP requests from master to peers instance when new artifact is saved then peers invoke artifact pooling into proxy repository.
+
+## Setting up plugin
+
+Plugin should be installed into master and peer Nexus servers both.
+
+### Master instance
+
+1. Download source from [https://github.com/griddynamics/nexus-replication-plugin](https://github.com/griddynamics/nexus-replication-plugin).
+2. Run `mvm package` in root directory (project requires Apache Maven 3.0.4 -- 3.1 and JDK 1.7+).
+3. Unzip the `target/nexus-replication-plugin-1.0-SNAPSHOT-bundle.zip` file into the plugin-repository directory (located in `$NEXUS_HOME/sonatype-work/nexus/plugin-repository`).
```

Comment:
I'd suggest to add `<finalName` directive to the POM so one won't need to take changing into account.